content/concepts/nat/autonat.md

```diff
@@ -1,27 +1,52 @@
 ---
 title: "AutoNAT"
-description: AutoNAT lets peers request dial-backs from peers providing the AutoNAT service.
+description: "AutoNAT lets peers request dial-backs from peers providing the AutoNAT service."
 weight: 200
 aliases:
     - "/concepts/autonat"
     - "/concepts/nat/autonat"
 ---
 
+## Background
+
+While the [identify protocol][spec_identify] lets peers inform each other about their observed network
+addresses, not all networks will allow incoming connections on the same port used for dialing out.
+
+To solve this problem, libp2p has implemented a protocol called AutoNAT, which allows nodes to determine
+whether or not they are behind a NAT and, if necessary, find a way to improve their connectivity.
```

Comment:
```suggestion
To prevent this problem of advertising undialable addresses, libp2p has implemented a protocol called AutoNAT, which allows nodes to determine
whether or not they are behind a NAT.
Furthermore, this lays the foundation for nodes to find a way to improve their connectivity to peers in public networks.
```